### ISSUE TITLE
Pi Node 4.11 doesn't recognize Docker desktop

### ISSUE THREAD

**Issue by shkdjsy** [pi-node/instructions#356]
when I turn on the node, stellar-dummy runs. No pi consensus starts
![pi node](https://github.com/pi-node/instructions/assets/169428673/01d559c2-5f89-438c-8a4e-11797af01631)

How to solve this?

**Comment by remzicirit** [comment]
I had same problem in teh past just remove docker and reinstall it solved the problem

**Comment by shkdjsy** [comment]
> I had same problem in teh past just remove docker and reinstall it solved the problem

I did many times. remove and reinstall both docker and pi node.

I only happen in this 4.11 version, any other version has this issue

**Comment by NET6666STREEM** [comment]
It seems all I am having is issues

**Comment by Rockman1955** [comment]
Same issues and same steps to correct, and still not showing he Pi-Consensus Container

**Please revise as to what happen in the Update?**

Thanks in advance

**Comment by Rockman1955** [comment]
> when I turn on the node, stellar-dummy runs. No pi consensus starts ![pi node](https://private-user-images.githubusercontent.com/169428673/329644393-01d559c2-5f89-438c-8a4e-11797af01631.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU0NDgxMzMsIm5iZiI6MTcxNTQ0NzgzMywicGF0aCI6Ii8xNjk0Mjg2NzMvMzI5NjQ0MzkzLTAxZDU1OWMyLTVmODktNDM4Yy04YTRlLTExNzk3YWYwMTYzMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMVQxNzE3MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNGViNjM1MDMxOGI0N2UzODE0NzMyZTgzMGE5YmI4NjhjNmU4NjA2N2Q1ZWM4NjJlZTcyYTUwZjY0YzdmZWY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.1-5vdIF-cy3QXYvhzclYNW2C_EyiSSkYcj9SmLkwu4M)
> 
> How to solve this?

Please update us all, and I will also if/we find the fix to the issue...

TIA

**Comment by Rockman1955** [comment]
> when I turn on the node, stellar-dummy runs. No pi consensus starts ![pi node](https://private-user-images.githubusercontent.com/169428673/329644393-01d559c2-5f89-438c-8a4e-11797af01631.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1NTI1MDMsIm5iZiI6MTcxNTU1MjIwMywicGF0aCI6Ii8xNjk0Mjg2NzMvMzI5NjQ0MzkzLTAxZDU1OWMyLTVmODktNDM4Yy04YTRlLTExNzk3YWYwMTYzMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQyMjE2NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MmM3ZjZlYWE5MjA3OTg3NDA3NTg5ZDAyMTQzNmQ1YWFjOGEwYjgyNTk1ZGI3YmM3YWM4NGZiMDIyM2ZjZmMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.6g0IaPHZC211AkKiA0jbC_Ol589fhsLTfxRMmfziCBc)
> 
> How to solve this?

Any Update or Fix to this Issues???

**Comment by openet001** [comment]
same  with issue 359 . tried to downgrade to pi node 0.4.7 ,  upgrade pi node 0.4.9 , pi node 0.4.11  are all ok.   But upgrade to pi node 0.4.11,  it seems  the bugs occurd again.

**Comment by Rockman1955** [comment]
> same with issue 359 . tried to downgrade to pi node 0.4.7 , upgrade pi node 0.4.9 , pi node 0.4.11 are all ok. But upgrade to pi node 0.4.11, it seems the bugs occurs again.

Yes same here, I uninstalled the 0.411 and re-installed the 0.49,
But when the 0.49 started, it then gave the message 'A new Version Is Available", and Pi Node will restart and install the new 0.411,

So back to where the 0.411 is not loading the Pi-Consensus Container....

Any other fix to this Issues??

**Comment by pabaldomar** [comment]
Same issue with macOs.

**Comment by NET6666STREEM** [comment]
Some machines don't have a virtual machine if you have a virtual machine make sure it's not connected to your system itself have it running off of your system like subsystem and make sure it's not connected to your PC whatsoever then when you run it install doctor within it I don't know if you would have to do pi node within it though I'm still trying to learn as much as I can before I start really doing my work and I have a lot of accumulated but it's going to give me a lot to do and I will have the time to do it so if anybody wants to get a head start be obliged but tag me in it thank you

**Comment by NET6666STREEM** [comment]
It's because of the point of like they told me I had to make another account use my link as far as my recommendation code for that account make sure it's me everything's correct just like the first one because of something I don't know but that is also for imaging examples learning purposes etc that will be figured out once main net is open net so any pie that I am accumulating within the other account only on this circumstance issued permission by CoreTeem 
Those are only circumstances that you should ever make another account you should never ever make another account only if you are an ambassador developer I repeat only if you are a ambassador developer thank you what I mentioned above doesn't apply to you then it doesn't apply to you okay no duplicate accounts unless ordered bye licensed Core Teem

**Comment by nhatvhm** [comment]
> Same issue with macOs.

Everything is automatically after you have done installation Pi Node v0.4.11 and Docker v4.31.0 on MacOS Monterey 12.7.5
Note: Please select configuration your Pi Node is not master if you don't have Port forwarding opening.

This is my Pi Node on Mac OS which my configuration is not master and still have Node bonus.
![GRtxxRubgAAiahA](https://github.com/pi-node/instructions/assets/7055239/18ad1947-73c5-4133-9e69-62b7cc41b56d)

**Comment by arnabnandy7** [comment]
I've tried installing and running Pi Node in Windows, but the issue remains. Do you have any ideas folks?

**Comment by Arielis01** [comment]
I need help with this, if someone can get it done for me remotly using teamviewer i will pay you as i have tried many times and i cant get it to run the node, my computer does not even recognize docker, ports are open my whatpp is 00 507 6815 5196

**Comment by dlisjak** [comment]
> I need help with this, if someone can get it done for me remotly using teamviewer i will pay you as i have tried many times and i cant get it to run the node, my computer does not even recognize docker, ports are open my whatpp is 00 507 6815 5196

Run the following command in the cmd or terminal:
MacOS: sudo ln -s /Applications/Docker.app/Contents/Resources/bin/docker /usr/local/bin/docker
Windows: mklink "C:\Path\to\symlink\docker.exe" "C:\Program Files\Docker\Docker\resources\bin\docker.exe"

**Comment by HeavyMedaI** [comment]
> > I need help with this, if someone can get it done for me remotly using teamviewer i will pay you as i have tried many times and i cant get it to run the node, my computer does not even recognize docker, ports are open my whatpp is 00 507 6815 5196
> 
> Run the following command in the cmd or terminal: MacOS: sudo ln -s /Applications/Docker.app/Contents/Resources/bin/docker /usr/local/bin/docker Windows: mklink "C:\Path\to\symlink\docker.exe" "C:\Program Files\Docker\Docker\resources\bin\docker.exe"

Worked, thanks.
